### ISSUE TITLE
<li> elements must be contained in an <ol> or <ul> element.

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2500]
## Builds

- [staging](https://console.cloud.google.com/cloud-build/builds;region=global/2df4af48-37fe-4d4f-8949-53c4b8cffaad?project=proudcity-1184)
- [master](https://console.cloud.google.com/cloud-build/builds;region=global/1a10a102-86b0-42b1-a027-1244a6f0a797?project=proudcity-1184)

## Source

https://airtable.com/appRFUc45sF4yWFut/tbl51inE74gEWlB0x/viwG9G4vAjsjQEh4g/rec4I1tKOMEBcWAvU?blocks=hide

## Describe the bug

`<li>` elements must be contained in an `<ol>` or `<ul>` element.  List item does not have a `<ul>`, `<ol>` parent element without a role, or a `role="list"`.  Currently the use of `"role="navigation"` on the UL's is obscuring their semantic role. `

## Recommendations

This occurs in the following locations.  Update the following sections:

1. The three skip links.  Remove the "Role=Navigation" from the `<ul>`.  This will allow a user to hear it is a list and that there are three items.   Add a `role="navigation"` to the preceding div or add the `<nav>` element around the whole list. 
2. The menu items in the navigation menu   Same solution as noted above. 
3. The Search link   Similar issue occuring with `role="toolbar"`

## Screenshots
![list 2](https://github.com/proudcity/wp-proudcity/assets/1569463/e0cbdeb5-2913-4855-825f-103372f4cd0d)
![lists 1](https://github.com/proudcity/wp-proudcity/assets/1569463/234b76ea-ca7e-44d6-a564-44b3361a54f5)

**Comment by curtismchale** [comment]
Deployed
Build: https://github.com/proudcity/proud-recipes/releases/tag/2024.05.02.1420x
Composer: https://github.com/proudcity/wp-proudcity/releases/tag/2024.05.02.1420
